test/test_functional.py

```diff
@@ -255,6 +255,7 @@ class Mytest(object):
             def test_hello(self, val):
                 pass
 
+        print(Mytest.__dict__)
```

Comment:
Can you remove this line? And I'll merge this. Thanks!

Reply:
Forgot to del that. Removed.